dashboard/lti_new.py

```diff
@@ -169,6 +169,12 @@ def extracting_launch_variables_for_tool_use(request, message_launch):
         'user_image': user_img,
         'primary_ui_color': settings.PRIMARY_UI_COLOR,
         'help_url': settings.HELP_URL,
+        'view_help_urls': { 
+            'ra':settings.URL_VIEW_RESOURCES_ACCESSED,
+            'apv1':settings.URL_VIEW_ASSIGNMENT_PLANNING_V1,
+            'ap':settings.URL_VIEW_ASSIGNMENT_PLANNING_V2,
+            'gd':settings.URL_VIEW_GRADE_DISTRIBUTION 
+        }  ,
```

Comment:
This file change may not be needed if this code get merged, https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1010